internal/version/version.go

```diff
@@ -0,0 +1,56 @@
+package version
+
+import (
+	"errors"
+	"runtime/debug"
+)
+
+var (
+	Version string
+)
+
+type VCSInfo struct {
+	VCS          string
+	Revision     string
+	Date         string
```

Comment:
I think that this can be renamed to `Time` so it is consistent with the info go is providing in the debug buildinfo package.
```suggestion
	Time         string
```